egs/heroico/s5/local/score_mbr.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
```

Comment:
this file is probably not needed -- the score_kaldi.sh can do mbr scoring as well (use the --use-mbr true switch, if needed)